src/list_items.graphql

```diff
@@ -1,73 +1,183 @@
-query ($projectId: ID!, $after: String) {
+query listItems($projectId: ID!, $after: String) {
   node(id: $projectId) {
-    ... on ProjectNext {
+    __typename
+    ... on ProjectV2 {
       items(first: 100, after: $after) {
         pageInfo {
           hasNextPage
           endCursor
         }
         nodes {
           id
-          title
           content {
             __typename
             ... on Issue {
+              title
               repository {
-                owner {
-                  login
-                }
-                name
+                nameWithOwner
               }
               number
-              labels(first: 100) {
+              labels(first: 10) {
```

Comment:
Why is it decreased? (is it hard-limitted to 10?)

Reply:
the old version will search up to about 1000000 nodes and will hit resource limitation:
https://docs.github.com/en/graphql/overview/resource-limitations#node-limit